scripts/vagrantCloudBasedContainers.sh

```diff
@@ -6,7 +6,7 @@
 # It builds a Docker image for GPII/universal and uses it to start two
 # components: the Preferences Server and the Flow Manager.
 #
-# It also starts a CouchDB container and loads the CouchDB data into
+# It also starts a CouchDB container and data into
```

Comment:
Old text was correct.

Reply:
Thanks for catching this, not sure how it got in and I didn't notice, reverted back to original text.